cypress/support/pages/user.pageObject.js

```diff
@@ -0,0 +1,25 @@
+import PageObject from '../PageObject';
+
+class userPageObject extends PageObject {
```

Comment:
The class name should be capitalized. Change 'userPageObject' to 'UserPageObject'.